packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -236,3 +236,104 @@ def get_task_url(dcos_api_session, task_name, stream=False):
     return '/system/v1/agent/{}/logs/v1/{}/framework/{}/executor/{}/container/{}'.format(slave_id, endpoint_type,
                                                                                          framework_id, executor_id,
                                                                                          container_id)
+
+
+def test_log_v2_text(dcos_api_session):
+    for node in dcos_api_session.masters + dcos_api_session.all_slaves:
+        response = dcos_api_session.logs.get('v2/component?limit=10', node=node)
+        check_response_ok(response, {'Content-Type': 'text/plain'})
+
+        # expect 10 lines
+        lines = list(filter(lambda x: x != '', response.content.decode().split('\n')))
+        assert len(lines) == 10, 'Expect 10 log entries. Got {}. All lines {}'.format(len(lines), lines)
+
+
+def test_log_v2_server_sent_events(dcos_api_session):
+    for node in dcos_api_session.masters + dcos_api_session.all_slaves:
+        response = dcos_api_session.logs.get('v2/component?limit=1', node=node, headers={'Accept': 'text/event-stream'},
+                                             stream=True)
+        check_response_ok(response, {'Content-Type': 'text/event-stream', 'Cache-Control': 'no-cache'})
+        lines = response.iter_lines()
+        sse_id = next(lines)
+        assert sse_id, 'First line must be id. Got {}'.format(sse_id)
```

Comment:
Would it make sense to check the content of the sse_id content since it's special?  Also, the same for the other stream test.

Reply:
ssd_id is a weird string similar to `s=cf866b2ad76146a1b3d0ef7c6a3c49ba;i=1;b=0cb30949e3514aa3864b9319fbd0f0f1;m=24de03f115;t=55e59f7f6c925;x=651ec921c9e375c4`, i am not sure how to validate it correctly.

Reply:
the purpose of this test is to make sure `sse_id` is not empty

Reply:
could you validate with a basic regex?